README.md

```diff
@@ -58,6 +58,15 @@ installers for all supported operating systems.
     sudo apt-get install etcher-electron
     ```
 
+##### Uninstall
+
+```sh
+sudo apt-get remove etcher-electron
+sudo apt-key del D401AB61
```

Comment:
In hindsight, I'm now not sure if the key-removal should be in the uninstall instructions. AFAIK it doesn't hurt to leave it in, and there _might_ be other packages on the user's system also installed from bintray?